erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md

```diff
@@ -30,16 +30,76 @@ Before creating and using a Quality Inspection, it is advised that you do the fo
 <img class="screenshot" alt="Quality Inspection" src="{{docs_base_url}}/assets/img/stock/quality-inspection.png">
 
 ## 3. Features
-### 3.1 Formula Based Quality Checks
-The acceptance of a parameter/check, in a Quality Inspection, can depend on a certain formula in many cases. The Readings table has a field called **Acceptance Criteria Formula** where you can specify a formula that determines whether a certain check is Accepted or Rejected.
+
+A single Quality Inspection consists of many Quality Checks (Parameters) within it. Each of these checks could be [Numeric](#31-numeric-quality-checks), [Non-numeric](#32-non-numeric-value-based-quality-checks) or [Formula Based](#33-formula-based-quality-checks).
+
+### 3.1 Numeric Quality Checks
+Numeric Quality Checks include all checks that require number based readings and acceptance criteria.
+
+E.g. checking if a reading is in a certain range.
+
+By default the checks are numeric. There are two fields: **Minimum Value** and **Maximum Value**, to define a range that **each** reading must be in. These fields can be set in the Quality Inspection Template once and be simply fetched into the Quality Inspection.
+
+<img class="screenshot" alt="Numeric Quality Check" src="{{docs_base_url}}/assets/img/stock/quality-inspection-numeric-reading.png">
+
+If any of the readings entered are not within this range, the status on the row will be set to 'Rejected' automatically on Save.
+
+### 3.2 Non-numeric (Value Based) Quality Checks
+Non-numeric Quality Checks include checks that require alphabetical values or those that do not require any mathematical calculations.
+
+E.g. checking if the color is white in a color quality check, Yes/No values for certain parameters, etc.
+
+For Non-numeric checks enable the 'Non-numeric' checkbox. You will notice a field **Acceptance Criteria Value** and a section **Value Based Inspection** are visible.
+
+You can simply enter the field Reading Value. The Acceptance Criteria Value can be set in the Quality Inspection Template once and be simply fetched into the Quality Inspection.
+
+<img class="screenshot" alt="Non-numeric Quality Check" src="{{docs_base_url}}/assets/img/stock/quality-inspection-non-numeric-reading.png">
+
+If the Reading Value does not match the Acceptance Criteria Value, the status on the row will be set to 'Rejected' automatically on Save.
+
+### 3.3 Formula Based Quality Checks
+Formula Based Quality Checks are useful for more complex scenarios where just specifying a range or an acceptance value is not enough.
+
+E.g. checking if the grade of a material is A/B/C, checking if the mean of some readings is in a certain range, etc.
```

Comment:
```suggestion
E.g. checking if the grade of a material is A/B/C, checking if the mean of some reading is within a certain range, etc.
```

Reply:
should be `readings`. Cant take mean of one reading